packages/quill/src/crypto/auth.sagas.ts

```diff
@@ -67,6 +76,37 @@ export function* createPasswordWorker({ payload }: PayloadAction<string>) {
   yield put(push(ROUTE_PATHS.SETUP_ACCOUNT));
 }
 
+export function* changePasswordWorker({ payload }: ReturnType<typeof changePassword>) {
+  try {
+    const isEqual: boolean = yield call(comparePassword, payload.currentPassword);
+    if (!isEqual) {
+      yield put(changePasswordFailed(translateRaw('CURRENT_PASSWORD_NOT_EQUAL')));
+      return;
+    }
+
+    const credentials: { account: string }[] = yield call(keytar.findCredentials, KEYTAR_SERVICE);
+    const privateKeys: [account: TUuid, privateKey: string][] = yield all(
```

Comment:
Does this not need a `call` somewhere or is it okay to pass Promises directly to `all`?

Reply:
AFAIK it's just the `redux-saga` way of doing `Promise.all` so I don't think this needs a `call`.

Reply:
Can't find anything about that in the docs 🤔 
https://redux-saga.js.org/docs/api/#alleffects---parallel-effects